sessionctx/variable/session.go

```diff
@@ -294,6 +294,9 @@ type SessionVars struct {
 	EnableStreaming bool
 
 	writeStmtBufs WriteStmtBufs
+
+	// L2CacheSize indicates the size of CPU L2 cache, using byte as unit.
+	L2CacheSize int
```

Comment:
We haven't set its value yet?

Reply:
No, the third point in the description describes this behavior.